[CUBRIDMAN-12] In data comparing, no ignoring to trailing spaces at the end of the data. (#206)
CUBRID/cubrid-manual: en/admin/config.rst

      
        1, NULL

**ignore_trailing_space**

> If this parameter's value is **yes**, the string comparsion follows "trailing space insensitive" rules as below examle.

Reviewer: examle --> example

Author: Ok, I fixed.

The string value 'abc' and 'abc ' is equal regardless the types of values are VARCHAR-type with variable length.

**CAUTION:**

> For the string data stored as "ignore_trailing_space" is set to **no** or **yes**, it will result to confuse at several parts of using the data when the configuration parameter value is changed to set to **yes** or **no**. Therefore, this configuration value change should be considered seriously. CUBRID recommends that the value of this parameter is fixed not to change for all life times in the database.

Reviewer: - For the string data stored as "ignore_trailing_space" is set to **no** or **yes**, it will result to confuse at several parts of using the data when the configuration parameter value is changed to set to **yes** or **no**.

--> The string data stored as "ignore_trailing_space" can be set to **no** or **yes**; will result in confusion at several parts of using the data when the configuration parameter value is changed between **yes** or **no**.

- CUBRID recommends that the value of this parameter is fixed not to change for all lifetimes in the database.

--> CUBRID recommends that the value of this parameter is fixed and not to change for all lifetimes in the database.

Author: Ok, I fixed.

> For the string data stored as **ignore_trailing_space** is set to **no** or **yes**; will result in confusion at several parts of using the data when the configuration parameter value is changed to set to **yes** or **no**. Therefore, this configuration value change should be considered seriously. CUBRID recommends that the value of this parameter is fixed and not to change for all life times in the database.

Reviewer 2: If it is a parameter that cannot be changed, how about making it hidden?

> For the string data stored as **ignore_trailing_space** is set to **no** or **yes**; will result in confusion at several parts of using the data when the configuration parameter value is changed to set to **yes** or **no**. Therefore, this configuration value change should be considered seriously. CUBRID recommends that the value of this parameter is fixed and not to change for all life times in the database.

Reviewer 3: I suggest the following changes.
1. "do not change the parameter value" needs to appear first. It is the main point of the caution.
2. "change" already has the meaning 'no to yes' and 'yes to no'.
3. "this parameter is fixed" seems to be ambiguous. it needs to give a specific moment.
4. need to give specific situations of what confusion or misbehavior is caused.

      
        1, NULL

**ignore_trailing_space**

If this parameter's value is **yes**, the string comparison follows "trailing space insensitive" rules as below example.
The string value 'abc' and 'abc ' is equal regardless the types of values are VARCHAR-type with variable length.

**CAUTION:**

> Do not change this parameter value or it will result in confusion at several parts of using the data. Especially, for the table with primary key or unique key using string type, the data integrity may not be preserved by changing this parameter value. For example, data with unique key constraint at **"no ignore trailing space"** configuration might violate this constraint at **"ignore trailing space"** because the key value 'abc  'and 'abc ' is equal at changed configuration. Therefore, this configuration value change should be considered seriously. CUBRID recommends that the value of this parameter is fixed and not to change for all life times in the database.

Reviewer 3: ```suggestion
You should not change this parameter value for all life times in the database. If not it will result in confusion at several parts of using the data. Especially, for the table with primary key or unique key using string type, the data integrity may not be preserved by changing this parameter value. For example, data with unique key constraint at **"no ignore trailing space"** configuration might violate this constraint at **"ignore trailing space"** because the key value 'abc  'and 'abc ' is equal at changed configuration. Therefore, this configuration value change should be considered seriously.
```
